package.json

```diff
@@ -18,7 +18,8 @@
     "next": "15.2.3",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
-    "react-icons": "^5.5.0"
+    "react-icons": "^5.5.0",
+    "wcn-website": "file:"
```

Comment:
Why do you keep adding this to the package.json. Is this needed?

Reply:
that ```"wcn-website": "file:"``` coming after refreshing running the site. Looking for the solution also

Reply:
@sunsplat got this one:https://shorturl.at/1tnbA

Should I try pruning the dependencies?

Reply:
I think you can just delete the line and run an npm install. It's weird because my package.json doesn't update that when I run the app. Maybe you have a setting in your code editor?